internal/bminventory/inventory.go

```diff
@@ -2342,6 +2342,10 @@ func (b *bareMetalInventory) ListClusters(ctx context.Context, params installer.
 		whereCondition += fmt.Sprintf(" AND openshift_cluster_id = '%s'", *params.OpenshiftClusterID)
 	}
 
+	if params.AmsSubscriptionIds != nil {
```

Comment:
if not given, it's ``nil`` or an empty list?

Reply:
The result is the same, but I will guess that it uses the go default value, meaning for `*string` we get `nil` and for array of string we get `string[]{}`.

So maybe worth changing that condition to `if len(params.AmsSubscriptionIds) > 0` just for aesthetics (so the DB query don't contains and "empty" condition that is always true).

Reply:
Changed.